app/services/discover/register.rb

```diff
@@ -0,0 +1,34 @@
+module Discover
+  class Register
+    FOREM_DISCOVER_URL = "https://discover.forem.com/api/forems/register".freeze
+
+    def initialize(domain:)
+      @domain = domain
+    end
+
+    def self.call(**args)
```

Comment:
Rubocop prefers class methods to come before instance methods. This is why the order is reverse in our service generator.